src/file-uploader/file-uploader.js

```diff
@@ -111,13 +112,9 @@ function FileUploader(props: PropsT) {
                         disabled={props.disabled}
                         kind={KIND.minimal}
                         onClick={open}
-                        overrides={{
```

Comment:
This is the right approach!
I believe `<Button>` should be replaced with `<ButtonComponent>` now. Also, the `Button` override needs to be specified in the types file here: https://github.com/uber-web/baseui/blob/master/src/file-uploader/types.js#L22 (I would give it a more descriptive title though)

Reply:
Done! I chose to specify it in the types file as `ButtonComponent`, mostly for consistency with the other overrides. Let me know if you want me to change this or something else. Thanks!